### PR TITLE
Allows gender neutral humans in preferences

### PR DESCRIPTION
**Pull request by Cenrus**
A port of https://github.com/tgstation/tgstation/pull/44530

## About this pull request

Restores gender neutral preferences from pre-base NSV.

## Why it's good for the game

Some people wanted it back.

## Changelog

:cl: Shaps/Ryll
tweak: NT has updated its personnel policies and allowed for employees to register as gender neutral, with standard support for they/them pronouns.
/:cl:

### CODE REVIEW

**Comment by Vasily2013** [comment]
What overall effect might this have on jobs such as genetics?

**Comment by Kmc2000** [comment]
This seems like a lot of code for this.

What's the purpose of this change anyway?

**Comment by Crossedfall** [comment]
> 
> 
> This seems like a lot of code for this.
> 
> What's the purpose of this change anyway?

Reading through the code it just allows for gender neutral options. I.e. instead of being male/female, you can be an other. Also replaces the instances of 'sex' with 'gender' to be more PC presumably.

File: code/game/objects/structures/mirror.dm [hunk 1]
```diff
@@ -28,7 +28,7 @@
 		//this is largely copypasted from there.
 
 		//handle facial hair (if necessary)
-		if(H.gender == MALE)
+		if(H.gender != FEMALE)
```
Comment (Crossedfall) [inline code/game/objects/structures/mirror.dm]: Seems needlessly swapped, but I guess if it works.

**Comment by Kmc2000** [comment]
BAD FRANCE

**Comment by Cenrus** [comment]
Won't change genetics in any way.

**Comment by Vasily2013** [comment]
> Won't change genetics in any way.

Have you tested it?

**Comment by francinum** [comment]
I'm not going to leave this PR to stagnate any larger, I don't particularly find the additional handling for this all that worth it at the moment. Possibly down the line, but not now.

**Pull request closed without merging**